src/app/dim-ui/Sheet.tsx

```diff
@@ -14,6 +16,8 @@ import { useDrag } from 'react-use-gesture';
 import { AppIcon, disabledIcon } from '../shell/icons';
 import './Sheet.scss';
 
+export const SheetContext = createContext<MutableRefObject<HTMLDivElement | null> | null>(null);
```

Comment:
I was a bit undecided to pass in the element or that ref object, I decided on the ref object because it is stable.